src-opam/dockerfile_distro.ml

```diff
@@ -229,7 +229,7 @@ let apt_opam ?pin ?opam_version ?compiler_version labels distro tag =
     Linux.Git.init () @@
     opam_init ?compiler_version ~need_upgrade () @@
     (match pin with Some x -> run_as_opam "opam pin add %s" x | None -> empty) @@
-    run_as_opam "opam install -y depext travis-opam" @@
+    Dockerfile.copy ~from:"ocaml/ci-opam" ~src:["/ci-opam"] ~dst:"/usr/bin" () @@
```

Comment:
does this really work?  I thought you had to specify a previous build stage, not an arbitrary other image...

Reply:
See https://docs.docker.com/develop/develop-images/multistage-build/#use-an-external-image-as-a-stage :-)